Sources/Lumina/Camera/Extensions/Delegates/PhotoCaptureDelegateExtension.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  PhotoCaptureDelegateExtension.swift
+//  Lumina
+//
+//  Created by David Okun on 11/20/17.
+//  Copyright © 2017 David Okun. All rights reserved.
+//
+
+import Foundation
+import AVFoundation
+
+extension LuminaCamera: AVCapturePhotoCaptureDelegate {
+  func photoOutput(_ output: AVCapturePhotoOutput, didFinishProcessingPhoto photo: AVCapturePhoto, error: Error?) {
+    LuminaLogger.notice(message: "finished processing photo")
+    guard let image = photo.normalizedImage(forCameraPosition: self.position) else {
+      return
+    }
+    photoCollectionQueue.sync {
+      if self.currentPhotoCollection == nil {
+        var collection = LuminaPhotoCapture()
+        collection.camera = self
+        collection.depthData = photo.depthData
+        collection.stillImage = image
+        self.currentPhotoCollection = collection
+      } else {
+        guard var collection = self.currentPhotoCollection else {
+          return
+        }
+        collection.camera = self
+        collection.depthData = photo.depthData
+        collection.stillImage = image
+        self.currentPhotoCollection = collection
+      }
+    }
+  }
+  
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dokun1/Lumina/blob/5e323e0c2296f8f5c5b041c942e35a2720d9c81e/Sources/Lumina/Camera/Extensions/Delegates/PhotoCaptureDelegateExtension.swift#L36"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>PhotoCaptureDelegateExtension.swift:36</code></td>
    </tr>
  </tbody>
</table>